api/python/quilt3/main.py

```diff
@@ -202,13 +203,18 @@ def cmd_verify(name, registry, top_hash, dir, extra_files_ok):
         return 1
 
 
-def cmd_push(name, dir, registry, dest, message, meta, workflow, force, dedupe):
+def cmd_push(name, dir_logical_key, dir, registry, dest, message, meta, workflow, force, dedupe, browse):
+    pkg = Package()
+
     try:
-        pkg = Package.browse(name, None)
+        pkg = Package.browse(name, browse)
```

Comment:
I assume you mean "registry=browse"? That would seem to imply the user has to pass the explicit registry to the browse option.
We were imagining something more like:
```suggestion
        browse_registry = (browse == "remote") ? registry : None
        pkg = Package.browse(name, registry=browse_registry)
```

Reply:
Also: could we do `Package.browse(name, registry=browse_registry)` AFTER the push to update the local registry, so it is possible to push twice in a row?

Reply:
Also, I am still unclear if (or when) browse() updates the local registry. How can/do we test that?

Reply:
@sir-sigurd You were correct. With these semantics, '--browse' it is identical to '--force' and therefore unnecessary. Please remove.

Reply:
Also, Aneesh confirmed that he mean literally "remote" not a registry name.